cmake/kokkos_arch.cmake

```diff
@@ -35,6 +35,7 @@ KOKKOS_ARCH_OPTION(ARMV80          HOST "ARMv8.0 Compatible CPU")
 KOKKOS_ARCH_OPTION(ARMV81          HOST "ARMv8.1 Compatible CPU")
 KOKKOS_ARCH_OPTION(ARMV8_THUNDERX  HOST "ARMv8 Cavium ThunderX CPU")
 KOKKOS_ARCH_OPTION(ARMV8_THUNDERX2 HOST "ARMv8 Cavium ThunderX2 CPU")
+KOKKOS_ARCH_OPTION(A64FX       HOST "ARMv8.2 with SVE Suport")
```

Comment:
Do not break the alignment.

Reply:
bah that came through the renaming from ARMV8_SVE to A64FX based on feedback from @nmhamster . Gonna fix that,

Reply:
fixed